3.6/oneshard.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: default
+description: queries using the OneShard feature can be faster
+---
+
+OneShard enterpise-feature
+==========================
+
+What is the OneShard feature?
+-----------------------------
+
+Under certain conditions big parts of the execution can be pushed to a single
+db-servers. This can result in big performance gains because usually multiple
+db-servers and a coordinator are involved in the execution of a query.
+
+Without the OneShard feature query processing works as follows. The coordinator
+accepts and analyses the query. If collections are accessed the coordinator
+distributes the accesses to collections to different db-servers that hold parts
+(shards) of the collections in question.
+This distributed access requires network-traffic from coordinator to dbservers
```

Comment:
`dbservers` vs. `db-servers`. There should be a consistent spelling.

Reply:
Yes, and I think we generally use `DBServers`. I think it would make sense to change this to `DB-Servers` however.